README.md

```diff
@@ -279,10 +211,10 @@ I would like to thank...
 
 <!-- LICENSE -->
 
-## 📝 License <a name="license"></a>
+## 📝 License <a name="license" href="https://github.com/Munyabelden/helloSetup/blob/hello-microverse/MIT.md">Mit.md</a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
[OPTIONAL] I suggest you remove the redundant lines above since they both point to the same thing. Line 216 looks neater and more professional so I suggest you retain that. You need to fix the link so that it points to your licence like so
`This project is [MIT](./MIT.md) licensed`